etc/profile-m-z/thunderbird.profile

```diff
@@ -8,9 +8,17 @@ include globals.local
 
 ignore include whitelist-runuser-common.inc
 
-# writable-run-user and dbus are needed by enigmail
+# TB stopped supporting enigmail in 2020 (v78) - let's harden D-Bus
+# https://support.mozilla.org/en-US/kb/openpgp-thunderbird-howto-and-faq
 ignore dbus-user none
-ignore dbus-system none
+dbus-user filter
+dbus-user.own org.mozilla.thunderbird.*
+dbus-user.talk ca.desrt.dconf
+dbus-user.talk org.freedesktop.Notifications
+# allow D-Bus communication with firefox for opening links
+dbus-user.talk org.mozilla.*
+# e2ee email needs writable-run-user
+# https://support.mozilla.org/en-US/kb/introduction-to-e2e-encryption
 writable-run-user
```

Comment:
What about wrutable-run-user?

Reply:
Not sure what you mean. I've tested the latest TB v115. End-to-end encryption is supported but users need to enable it and TB can create keys for each of their accounts. Encrypting and signing email does need `writable-run-user`. That didn't change by dropping enigmail support and integrating OpenPGP and S/MIME into TB. The same goes for claws-mail for example, which is why our email-common.profile has writable-run-user too.

Is there anything else you wanted to point out regarding wru?

Reply:
Was just asking because of the old comment.

Reply:
Ow okay. No problem.